disnake/flags.py

```diff
@@ -527,10 +529,8 @@ def __init__(self, **kwargs: bool):
     @classmethod
     def all(cls: Type[Intents]) -> Intents:
         """A factory method that creates a :class:`Intents` with everything enabled."""
-        bits = max(cls.VALID_FLAGS.values()).bit_length()
-        value = (1 << bits) - 1
         self = cls.__new__(cls)
-        self.value = value
+        self.value = functools.reduce(operator.or_, cls.VALID_FLAGS.values())
```

Comment:
Why are we importing an entire module just to avoid a lambda expression? Or even `int.__or__`

Reply:
`operator.or_` appears to be around 50% faster than `int.__or__` in cpython for some reason, although performance really doesn't matter at all here since it's expected to only be called a few times at most.

```py
In [3]: d = tuple(...)  # some random numbers

In [4]: %timeit functools.reduce(operator.or_, d)
41.5 µs ± 571 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)

In [5]: %timeit functools.reduce(int.__or__, d)
61.9 µs ± 511 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```

Reply:
This is interesting